### ISSUE TITLE
pvl_dirint.py

### ISSUE THREAD

**Issue by Jorissup** [Sandia-Labs/PVLIB_Python#40]
Just a question, 

Do you plan on making pvl_dirint.py, I only see the matlab version?

**Comment by Calama-Consulting** [comment]
Yes, that one slipped through the cracks, it should be up in the next few weeks

**Comment by Jorissup** [comment]
Thanks,

Only asking because when I use the Disc model it seems to give the wrong output?
Only when I change line 105:  DNI=(Kn)*(I0)
to use the horizontal radiation e.g: DNI=(Kn)*(I0h)
I seem to get a better output

**Comment by Lamaf** [comment]
I am also having wrong output ... for example try this

    import datetime
    import pandas as pd
    from pvlib.location import Location
    import pvlib.solarposition
    import pvlib.clearsky
    
    # make a location
    tus = Location(32.2, -111, 'MST', 700)
    # make a pandas DatetimeIndex for some day
    times = pd.date_range(start=datetime.datetime(2014,6,24), end=datetime.datetime(2014,6,25), freq='1Min')
    # calculate the solar position
    solpos = pvlib.solarposition.get_solarposition(times, tus, method='pyephem')
    # calculate clear sky data
    tus_cs = pvlib.clearsky.ineichen(times, tus, airmass_model='young1994')
    ax = tus_cs.plot()
    # calculate the direct normal irradiance using GHI from the clear sky data
    DNI= pvlib.clearsky.disc(Time= solpos.index, GHI=tus_cs.GHI, SunZen=solpos.zenith)
    DNI.DNI_gen_DISC.plot(ax=ax, legend=True)

Why do we have missing values when using DISC to simulate DNI?
Am I doing something wrong ??

![dni](https://cloud.githubusercontent.com/assets/11089685/6285775/0d132624-b900-11e4-8aa4-508943c02f00.png)

**Comment by wholmgren** [comment]
@Lamaf do you still have this issue if you use the new official package at 

https://github.com/pvlib/pvlib-python

If so, please make a new issue at that repository, and a PR including new unittests would be greatly appreciated.

**Comment by Lamaf** [comment]
@wholmgren Yes I am using the official package, and I opened a new issue with the subject, thanks for the reply.
